front/src/Components/Footer/index.jsx

```diff
@@ -4,53 +4,55 @@ import LogoInThePear from '../../assets/LogoInThePear.png';
 import LogoInThePearResponsive from '../../assets/LogoInThePearResponsive.png';
 import { BsInstagram } from 'react-icons/bs';
 import { BsFacebook } from 'react-icons/bs';
-import './style.css';
+import './styles.css';
 
-const Footer = () => (
+const Footer = () => {
```

Comment:
```suggestion
const Footer = () => (
```
como es un componete presentacional , o sea sin logica, no debe llevar llevas